<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameFrame</class>
 <widget class="QFrame" name="GameFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameFrame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="forceVectorLabel">
       <property name="text">
        <string>Force</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="readyPushBtn">
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="speedLabel">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="6">
      <widget class="QLCDNumber" name="speedLcdNumber">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>4</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="forceLcdNumber">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>4</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
